tools/testsys/src/delete.rs

```diff
@@ -0,0 +1,27 @@
+use anyhow::{Context, Result};
+use clap::Parser;
+use futures::TryStreamExt;
+use model::test_manager::{DeleteEvent, TestManager};
+
+/// Delete all objects from a testsys cluster.
```

Comment:
The word "resource" is unfortunate here, but the documentation should not imply that, e.g. the controller or any other things will be deleted.

```suggestion
/// Delete all tests and resources from a testsys cluster.
```